package.json

```diff
@@ -40,6 +42,13 @@
     "test-php": "wp-env run phpunit 'phpunit -c /var/www/html/wp-content/plugins/performance/phpunit.xml.dist --verbose'",
     "pretest-php-multisite": "wp-env run composer 'install --no-interaction'",
     "test-php-multisite": "wp-env run phpunit 'WP_MULTISITE=1 phpunit -c /var/www/html/wp-content/plugins/performance/tests/multisite.xml --verbose'",
-    "wp-env": "wp-env"
+    "wp-env": "wp-env",
+    "prepare": "husky install"
```

Comment:
Wondering the `prepare` as I don't think we are actually packaging the plugin as a new version.

Reply:
Where is `prepare` being run?

Reply:
I guess it would be more useful a precommit hook with husky that runs test lint and so on
https://khalilstemmler.com/blogs/tooling/enforcing-husky-precommit-hooks/#Configuring-Husky

Anyhow this is the same husky setup used in the Gutenberg repo, which is fine.

Reply:
@mitogh @dainemawer `prepare` is a [lifecycle event](https://docs.npmjs.com/cli/v8/using-npm/scripts#life-cycle-scripts) in NPM. [Here](https://docs.npmjs.com/cli/v8/using-npm/scripts#npm-install) is the sequence in which events are run in when running `npm install`. I have put `husky install` in package.json as [mentioned in](https://typicode.github.io/husky/#/?id=install) husky's docs. 

Basically, we need to run `husky install` once after cloning the repo. [What it does is](https://stackoverflow.com/a/66786333/2867422), add the `.husky` directory to git's `core.hooksPath`